.circleci/config.yml

```diff
@@ -541,3 +544,17 @@ jobs:
       - run:
           name: "Run ubuntu-cpp image"
           command: docker-compose run -e NUM_THREADS=16 --rm ubuntu-cpp
+
+  docker-compose-ubuntu-adapters-cpp:
+    machine: # Use machine executor type to use docker-compose
+      image: ubuntu-2204:2022.10.1 # Ubuntu 22.04
+    resource_class: 2xlarge
+    steps:
+      - checkout
+      - update-submodules
+      - run:
+          name: "Build images for ubuntu-adapters-cpp declared in docker-compose.yml"
+          command: docker-compose build ubuntu-adapters-cpp
```

Comment:
If you add `ubuntu-adapters-cpp.dockerfile` to the docker workflow you can just use the image from ghcr.io instead of building it (as that is updated on each push to master it should be recent enough, if there are any changes `run` should rebuild it anyway.

Reply:
Actually probably not true, it looks like exporting the cache manifest has to be explicitly enabled even with the `docker/build-push` action. I'll take care of that in a follow up.

Reply:
@assignUser I am resolving this comment as I understand this will be done on a follow  up PR. Let me know if you want me to do any changes.